aunt-leahs-app/src/components/SelectBox.jsx

```diff
@@ -1,44 +1,37 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { InputLabel, MenuItem, FormControl, Select } from '@material-ui/core';
 
-class SelectBox extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            items: this.props.items || [],
-            selectedItem: ''
-        };
-        this.handleSelect = this.handleSelect.bind(this);
-    }
-    handleSelect = (e) => {
-        this.props.onSelectItem(e.target.value)
-        this.setState({ selectedItem: e.target.value });
+const SelectBox = ({ name, items, onSelectItem }) => {
+
+    const [selectedItem, setSelectedItem] = useState('');
+
+    const handleChange = event => {
+        onSelectItem(event.target.value);
+        setSelectedItem(event.target.value);
     }
 
-    render() {
-        return (
-            <div>
-                <FormControl fullWidth={true}>
-                    <InputLabel id='volunteer-select-label'>{this.props.name || 'Select'}</InputLabel>
-                    <Select
-                        labelId='volunteer-select-label'
-                        id='my-select'
-                        value={this.state.selectedItem}
-                        onChange={this.handleSelect}
-                    >
-                        <MenuItem value=''>
-                            <em>None</em>
+    return (
+        <div>
+            <FormControl fullWidth={true}>
+                <InputLabel id='volunteer-select-label'>{name || 'Select'}</InputLabel>
+                <Select
+                    labelId='volunteer-select-label'
+                    id='my-select'
```

Comment:
I think all id's are supposed to be unique so having one for the generic component might lead to some undesirable effects. If this is being used for CSS, I think className works better here. (Also, same for `labelId`?)